src/pinmapping.h

```diff
@@ -25,8 +25,8 @@
 #define PIN_PRESSURESENSOR 36
 #define PIN_WATERSENSOR 23
 #define PIN_FLOWSENSOR 1
-#define PIN_HXDAT 32            // Brew scale data pin
-#define PIN_HXCLK 33            // Brew scale clock pin
+#define PIN_HXDAT 35            // Brew scale data pin
```

Comment:
I think this PR should not change the pin mapping. Until I changed this back, it did not work: 

```
E (7555) gpio: GPIO can only be used as input mode
[  7569][E][esp32-hal-gpio.c:130] __pinMode(): GPIO config failed
E (7558) gpio: gpio_set_level(226): GPIO output gpio_num error
```

Even if it did not output errors, that mapping clashes with the brew switch and steam switch defined above.

Reply:
The pinmapping is pretty old, back when the ESP8266 was the standard, but I wanted to use an ESP32. I'm still using it solo without the clevercoffee board.
This has to be adapted to the actual Pinmapping.